src/main/resources/org/biouno/unochoice/stapler/unochoice/unochoice.js

```diff
@@ -636,6 +644,10 @@ var UnoChoice = UnoChoice || (function($) {
             var filterElement = _self.getFilterElement();
             var filteredElement = _self.getParameterElement();
             var text = filterElement.value.toLowerCase();
+            if (text.length != 0 && text.length < _self.getFilterLength()) {
+                console.log("Filter pattern to short " + text.length + " < " + _self.getFilterLength());
```

Comment:
While testing, noticed that I got one message like this for each character entered. Could you remove this line @klimas7 ? In the future, we can discuss a way to enable/disable via Jenkins UI debug in the JS console. But for now let's limit what we put there (we are going to reduce even more I believe for JENKINS-34188).